cmd/flags.go

```diff
@@ -105,5 +105,10 @@ func CreateFlags(defaultPath string) []cli.Flag {
 			Name:  "pem",
 			Usage: "Generate a .pem file by concatenating the .key and .crt files together.",
 		},
+		cli.IntFlag{
```

Comment:
Why not just use the `DurationFlag` instead?

https://godoc.org/github.com/urfave/cli#DurationFlag